analysers/analyser_merge_power_substation_minor_FR.py

```diff
@@ -59,6 +59,6 @@ def __init__(self, config, logger = None):
                         "substation": "minor_distribution"},
                     mapping2 = {
                         "operator": "NOM_GRD",
-                        "source": self.source+"/"+fields["NOM_GRD"],
+                        "source": lambda fields: self.source+"/"+fields["NOM_GRD"],
```

Comment:
Better use a standard `;`, doesn't ?

Reply:
Not aware of any other mean to use a field in mapping expression. What are you referring to?

Reply:
`source=foo;bar` and not `source=foo/bar`

Reply:
Oh well.
They're not separate sources, let's say consortium :)

Reply:
Ok. Replaced by " - ".